kahuna/package.json

```diff
@@ -32,6 +32,7 @@
       "text": "github:systemjs/plugin-text@^0.0.2",
       "theseus": "npm:theseus@0.5.2",
       "theseus-angular": "npm:theseus-angular@^0.3.1",
+      "titip": "github:quantumui/titip@master",
```

Comment:
> Could you open an issue on that project to request them to tag versions?

https://github.com/quantumui/titip/issues/1

> is it possible to use Git hashes?

Yes it is. What would you advise for [jspm's registry](https://github.com/jspm/registry/pull/689)?